Drop - Deploy a local Ubuntu server hosted with Vagrant and connect it to Azure Arc

mrhoads opened the pull request:
this PR converts the Arc-enabled Server scenario for [deploying a local Ubuntu server hosted with Vagrant and connect it to Azure Arc](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vagrant/local_vagrant_ubuntu)

It's updated to work with multiple providers (e.g. Hyper-V, VirtualBox, libvirt, etc.)

fcabrera23 commented on drops/arc_server_deploy_vagrant.json:
```diff
@@ -0,0 +1,24 @@
+      "Arc",
+      "Arc-enabled Servers"
+    ],
+    "LastModified": "",
```
Ensure to add LastModified and CreatedDate dates

fcabrera23 commented on drops/arc_server_deploy_vagrant.json:
```diff
@@ -0,0 +1,24 @@
+    "ProgrammingLanguage": [
+        "Bash"
+    ],
+    "Products": [
```
Maybe add ubuntu?

fcabrera23 replied:
Maybe also add Vagrant?

fcabrera23 commented on drops/arc_server_deploy_vagrant.json:
```diff
@@ -0,0 +1,24 @@
+    "Description": "This Jumpstart Drop provides a step-by-step guide to create an Ubuntu 22.04 server with Vagrant and connecting it to Azure Arc.",
+    "Authors": [
+      {
+        "Name": "Mike Rhoads",
```
Who was the original owner? Maybe we could add him/her?

fcabrera23 reviewed (changes requested):
@mrhoads  could you please review?

mrhoads commented:
I added:

- Ubuntu and Vagrant as products
- timestamp for LastModified and CreatedDate
- changed Author section to Lior Kamrat based on Git history

fcabrera23 reviewed (approved):
LGTM
